packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -275,6 +275,22 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('setZoomLevel() is not implemented.');
   }
 
+  /// Gets a list of video stabilization modes that are supported for the selected camera.
+  Future<Iterable<VideoStabilizationMode>> getSupportedVideoStabilizationModes(
+      int cameraId) async {
+    throw UnimplementedError(
```

Comment:
This needs to return an empty list, so that the default behavior for implementations that don't support stabilization is gracefully reporting that, rather than throwing.